### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by Chung-Yi** [sergiomsilva/alpr-unconstrained#154]
I want to ask that my least training loss is between 0.8-1.5. Does anyone know how to decrease the loss. Although it has nice performance at most condition.
